plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1008,7 +1007,7 @@ func (c *PodAffinityChecker) InterPodAffinityMatches(pod *v1.Pod, meta interface
 // TODO: Do we really need any pod matching, or all pods matching? I think the latter.
 func (c *PodAffinityChecker) anyPodMatchesPodAffinityTerm(pod *v1.Pod, allPods []*v1.Pod, node *v1.Node, term *v1.PodAffinityTerm) (bool, bool, error) {
 	if len(term.TopologyKey) == 0 {
-		return false, false, errors.New("Empty topologyKey is not allowed except for PreferredDuringScheduling pod anti-affinity")
+		return false, false, fmt.Errorf("Empty topologyKey is not allowed except for PreferredDuringScheduling pod anti-affinity")
```

Comment:
fmt.Errorf almost equals errors.New.  The whole file use fmt.Errof except here. We should unify the format and style.
As to "capitalized", I am not sure. Although it shoud be low-case in error message, it seems not the style of k8s.